flang/lib/Optimizer/Transforms/CMakeLists.txt

```
@@ -46,7 +46,7 @@ add_flang_library(FIRTransforms
   FIRDialect
   FIRDialectSupport
   FIRSupport
-  FortranCommon
+  FortranSupport
   FortranSupport
```

This is now duplicated
```suggestion
  FortranSupport
```

---

Some auto-merge problem. fixed.